content/developer/howtos/translations.rst

```diff
@@ -63,8 +63,6 @@ Odoo automatically exports translatable strings from "data"-type content:
   * if ``selection`` is present and a list (or tuple), it's exported
   * if their ``translate`` attribute is set to ``True``, all of their existing
     values (across all records) are exported
-* help/error messages of :attr:`~odoo.models.Model._constraints` and
-  :attr:`~odoo.models.Model._sql_constraints` are exported
```

Comment:
Unless the error messages of constraints are not exported anymore, I believe it'd be better to keep a point about it :thinking:

Reply:
I just tried exporting translations for base and I see (random example).
```
#. module: base
#: model:ir.model.constraint,message:base.constraint_ir_model_obj_name_uniq
msgid "Each model must have a unique name."
```

It is true that the export is not done from the code anymore, but the table containing the translations of constraints is exported.

@HydrionBurst Could you confirm?

Reply:
- `_constraints` does not exist (already today) - translating is done in the method implementing `@api.constrains`
- `_sql_constraints` does not exist (this PR) - translating is done using the translated field on the reflection of the constraints (model: ir.model.constraints)